internals/serve-with-csp.json

```diff
@@ -0,0 +1,15 @@
+{
+  "public": "./build",
+  "headers": [
+    {
+      "source": "**",
+      "headers": [
+        { "key": "Comment", "value": "Keep Content-Security-Policy synced with deployment" },
+        {
+          "key": "Content-Security-Policy",
+          "value": "default-src 'none'; script-src 'report-sample' 'self'; style-src 'report-sample' 'self' fonts.googleapis.com 'unsafe-inline'; font-src 'self' fonts.gstatic.com; connect-src 'self' grpc.oasis.dev testnet.grpc.oasis.dev monitor.oasis.dev; img-src 'self'; base-uri 'self'; manifest-src 'self';"
```

Comment:
If this changes, ping @tjanez to update deploy

Reply:
Should we use
```
img-src 'self' https:
```
to have validator external icons still working?

Reply:
Added https: and data:

Reply:
ping @tjanez